draft-ietf-tsvwg-multipath-dccp.mkd

```diff
@@ -1429,6 +1429,58 @@ A DCCP implementation maintains the maximum packet size (MPS) during operation o
 
 For compatibility reasons, an MP-DCCP implementation SHOULD always announce the minimum MPS across all paths.
 
+## Path usage strategies
+
+MP-DCCP can be configured to realise one of several strategies for path usage, via selecting one DCCP subflow of the multiple DCCP subflows within a MP-DCCP connection for data transmission. This can be a dynamic process further facilitated by the means of DCCP and MP-DCCP defined options such as path preference using MP-PRIO, adding or removing DCCP subflows using MP_REMOVEADDR, MP_ADDADDR or DCCP-Close/DCCP-Reset and also path metrics such as packet-loss-rate, CWND or RTT provided by the Congestion Control Algorithm.
+Selecting an appropriate method can allow MP-DCCP to realise different path utilization strategies that make MP-DCCP suitable for end-to-end implementation over the Internet or in controlled environments such as Hybrid Access or 5G ATSSS.
+
+### Path mobility
+
+The path mobility strategy provides the use of a single path with a seamless handover function to continue the connection when the currently used path is deemed unsuitable for service delivery.
+Some of the DCCP subflows of a MP-DCCP connection might become inactive due to either the occurrence of certain error conditions (e.g., DCCP timeout, packet loss threshold, RTT threshold, closed/removed) or adjustments from the MP-DCCP user.
+When there is outbound data to send and the primary path becomes inactive (e.g., due to failures) or de-prioritized, the MP-DCCP endpoint SHOULD try to send the data through an alternate path with a different source or destination address (depending on the point of failure), if one exists. This process SHOULD respect the path prio configured by MP_PRIO or if not available pick the most divergent source-destination pair from the original used source-destination pair.
+Note: Rules for picking the most appropriate source-destination pair are an implementation decision and are not specified within this document.
+Path mobility is supported in the current Linux reference implementation {{website}}.
+
+### Concurrent path usage
+MP-DCCP does not specify the scheduling functions that determines when 
+or how the multiple subpaths are used to convey DCCP Data. Compared to a 
+path mobility strategy, the selection between MP-DCCP subflows is a 
+per-packet decision that is a part of the multipath scheduling process.
+
+The method allows a scheduling design to be used to realise various 
+designs of multipath usage.
+It can support path failover, where multiple subpaths remain active, but 
+only one subpath is used at any one time. Scheduling could seek to 
+maximise reliability, throughput, minimising latency, etc.
```

Comment:
```suggestion
The method allows a scheduling design to be used to realise various 
designs of multipath usage.
It can support path failover, where multiple subpaths remain active, but 
only one subpath is used at any one time. Scheduling could seek to 
maximise reliability, throughput, minimising latency, etc.
```